components/CustomInput.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Input } from 'galio-framework';
+
+const CustomInputText = props => {
+  const {
+    placeholder,
+    name,
+    defaultValue,
+    handleInputChange,
+    borderStyle,
+  } = props;
+  return (
+    <Input
+      placeholder={placeholder}
+      name={name}
+      defaultValue={defaultValue}
+      onChangeText={text => {
+        handleInputChange(text, name);
+      }}
+      style={borderStyle}
+      color="black"
+      rounded
+    />
+  );
+};
+
+const CustomInputNumber = props => {
+  const {
+    placeholder,
+    name,
+    defaultValue,
+    handleInputChange,
+    borderStyle,
+  } = props;
+  return (
+    <Input
```

Comment:
Why do we need three different components for text, password, number?
Can we use only one component for all the three types of input?